gitpractice/task2/src/app/constantfile.ts

```diff
@@ -0,0 +1,24 @@
+    export namespace AppConstants {
+    
+    export const HOMELINK= 'Home';
+    export const ACTIVELINK= 'active';
+    export const DELETELINK= 'delete';
+    export const MANAGELINK= 'Manage';
+    export const H1  ='Users Management System';
+    export const H2= 'You can choose active ,deleted and manage.';
```

Comment:
give proper names for the constants, they must be explaining the text which it contains